internal/testutils/mock/mock_config.go

```diff
@@ -2,6 +2,8 @@ package mock
 
 import "github.com/fabric8-services/fabric8-jenkins-idler/internal/util"
 
+// Config contains all configuration detail (prerequisite as well) to run idler.
```

Comment:
"Cofig is a mock implementation of the configuration.Configuration interface. It can be used in tests where any field can be explicitly set to return the needed value"